frontend/src/Pages/AdminPage.js

```diff
@@ -28,14 +32,30 @@ export default function AdminPage() {
     return wrongAnswers.length
   }
 
+  const howMany = answers => {
+    const users = answers.map(answer => answer.userName)
+    console.log(users)
+    const number = [...new Set(users)]
+    console.log(number)
+    return number.length
+  }
+
   useEffect(() => {
+    getAllUsers(token)
+      .then(response => response.data)
+      .then(data => {
+        const users = data.map(dat => dat.id)
+        setAllUsers(users.length)
+      })
+      .catch(e => setError(e))
     getStats(token)
       .then(response => response.data)
       .then(data => {
```

Comment:
find a better naming instead of data. I know it is the data from the request ;)

Reply:
Found it, fixed it 👍